src/tools/web/elektrad/README.md

```diff
@@ -1,30 +1,25 @@
 # elektra-web/elektrad
 
-A server that provides an [HTTP API](http://docs.elektrad.apiary.io/) to access
-Elektra remotely, built using [express](https://expressjs.com/).
+A server that provides an [HTTP API](http://docs.elektrad.apiary.io) to access
+Elektra remotely, built using [go](https://golang.org).
 
 ## Installation
 
-- first, install [libelektra](https://libelektra.org/)
-- now run `npm install`
+- make sure you have go (>1.11) installed.
+- install [libelektra](https://libelektra.org/), make sure the package-config env variable is set correctly.
```

Comment:
Which package-config env variable?

Reply:
I removed this comment and linked to the README of go-elektra which already contains troubleshooting documentation